src/containers/BookForm.js

```diff
@@ -16,23 +16,27 @@ class BooksForm extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      title: "",
+      key: Math.floor(Math.random() * 100),
+      input: "",
       category: bookCategories[0]
     };
     this.handleChange = this.handleChange.bind(this);
     this.handleSubmit = this.handleSubmit.bind(this);
   }
 
   handleChange(event) {
-    this.setState({ [event.target.id]: event.target.value });
+    this.setState({ [event.target.name]: event.target.value });
   }
 
   handleSubmit(event) {
     event.preventDefault();
-    const newBook = this.state;
-    const { createBook } = this.props;
-    createBook(newBook);
-    this.setState({ title: "", category: bookCategories[0] });
+    const newBook = {
+      key: new Date().getTime(),
+      title: this.state.input,
+      category: this.state.category
+    };
+    this.props.createBook(newBook);
+    return newBook;
   }
```

Comment:
After creating a new book, let's reset the state.

![image](https://user-images.githubusercontent.com/36057474/72072184-ae3b9600-32ed-11ea-91aa-e2638ab27da1.png)